Sources/MapLibreSwiftUI/Examples/Camera.swift

```diff
@@ -10,17 +10,18 @@ struct CameraDirectManipulationPreview: View {
 
     var body: some View {
         MapView(styleURL: styleURL, camera: $camera)
-            .overlay(alignment: .bottomLeading, content: {
+            .overlay(alignment: .bottom, content: {
                 switch camera {
                 case .centerAndZoom(let coord, let zoom):
-                    Text("\(coord.latitude), \(coord.longitude) z\(zoom ?? 0)")
-                        .padding(.all, 8)
-                        .background(in: .rect(cornerRadii: .init(topLeading: 8,bottomLeading: 8,bottomTrailing: 8,topTrailing: 8)), fillStyle: .init())
-                        .safeAreaPadding(.all)
+                    Text("\(coord.latitude), \(coord.longitude) z \(zoom ?? 0)")
+                        .padding()
+                        .background(in: RoundedRectangle(cornerRadius: 8), 
+                                    fillStyle: .init())
+                        .padding(.bottom, 42)
                 }
             })
         .task {
-            try! await Task.sleep(for: .seconds(3))
+            try! await Task.sleep(nanoseconds: 3_000_000_000)
```

Comment:
Major bummer we have to loose this; I forgot it didn't exist in iOS 15!

In the interim, maybe we could multiply by a constant (I think `NSEC_PER_SEC` is still a thing somewhere) to make it clearer what's going on, because I never remember how many nanoseconds are in a second myself.

Reply:
Sorted 👍